packages/components/src/VirtualizedList/NoRows/NoRows.component.js

```diff
@@ -6,20 +6,30 @@ import { getDefaultTranslate } from '../../translate';
 
 import theme from './NoRows.scss';
 
-function NoRows({ t }) {
+function getLabel(props) {
+	if (props.noRowsLabel !== undefined) {
```

Comment:
I think that overriding the i18n key when necessary could be more "i18n compliant"

Reply:
I have two list on one same component with 2 different labels each others when the list is empty. I can't do with the i18n because the same label 'll be applied in the 2 list.

Reply:
I prefer use a label to custom this by the application. This is more flexible if we have use more features from i18n (interpolation, ...)